src/main/kotlin/inc/evil/d17map/dtos/ReservationRequest.kt

```diff
@@ -3,33 +3,53 @@ package inc.evil.d17map.dtos
 import com.fasterxml.jackson.annotation.JsonFormat
 import inc.evil.d17map.enums.ReservationType
 import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.constraints.*
 import java.time.LocalDate
 import java.time.LocalTime
 import java.util.*
 
 data class ReservationRequest(
+    @field:NotBlank(message = "Title must not be blank.")
     val title: String,
+
+    @field:NotBlank(message = "Description must not be blank.")
     val description: String,
 
     @Schema(type = "string", format = "date")
     @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+    @field:NotNull(message = "Date must not be null.")
```

Comment:
Wydaje mi się, że kotlin uniemożliwia wpisanie nulla, ale do sprawdzenia (np. Chatgpt)

Reply:
Sprawdziłam, faktycznie nie jest to konieczne dla typu LocalDate - w takim razie usuwam